packages/jade/plugin/handler.js

```diff
@@ -47,6 +82,20 @@ var getCompilerResult = function (compileStep, fileMode) {
   }
 };
 
+// Generate helper functions using function name - function body map
+//   in the input.
+var templateHelperGen = function (helpers, tplName) {
```

Comment:
I could, but I would need to pass a generator function to create a higher order function, which would make it harder to read. Still, I did some refactoring to make handler.js simpler.